locales/en/settings.json

```diff
@@ -68,7 +68,7 @@
     "enterNumber": "Please enter a number between 0 and 24",
     "fillAll": "Please fill out all fields",
     "passSuccess": "Password successfully changed",
-    "usernameSuccess": "Username successfully changed",
+    "usernameSuccess": "Login Name successfully changed",
     "data": "Data",
-    "exportData": "Export Data"
-}
+    "exportData": "Export Data",
+    "emailChange": "To change your email address, please send an email to admin\@habitrpg.com including both your old and new email address as well as your User ID."}
```

Comment:
If you could make the email address a mailto: link, that would be very helpful for the users! See how other `a href` links are done in this file.